CRM/MembershipExtras/Service/UpfrontInstalments/AbstractUpfrontInstalmentsCreator.php

```diff
@@ -72,6 +72,7 @@ private function setLastContribution() {
         'fee_amount', 'total_amount', 'payment_instrument_id', 'is_test',
         'tax_amount', 'contribution_recur_id', 'financial_type_id', 'receive_date',
       ],
+      'is_test' => $this->currentRecurContribution['is_test'],
```

Comment:
check if $this->currentRecurContribution['is_test'] exists first and if it does not then force it to be 0

Reply:
Updated, please see the [CIWEMB-345: Handle payment plan in test mode](https://github.com/compucorp/uk.co.compucorp.membershipextras/pull/484/commits/235a1fcd4005917b1b172dfc3b9ced73a91e414e) commit.